src/main/scala/org/dia/algorithms/mcc/MCSUtils.scala

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dia.algorithms.mcc
+
+import java.util
+import scala.collection.mutable.ListBuffer
+import ucar.ma2.{ArrayInt, ArrayDouble, DataType}
+import ucar.nc2.{NetcdfFileWriter, Dimension, Attribute, Variable }
+import scala.reflect.ClassTag
+import scala.collection.JavaConversions._
```

Comment:
Try using scala.collection.JavaConverters._ instead.
This forces users to use .asScala and .asJava when converting equivalent collections across the languages.
The implicit conversions that JavaConversions does can be a tad too much magical to trace.